code/workspaces/client-api/src/schema/resolvers.js

```diff
@@ -26,7 +26,7 @@ const PROJECT_KEY = 'project';
 const resolvers = {
   Query: {
     status: () => () => `GraphQL server is running version: ${version}`,
-    dataStorage: (obj, args, { user, token }) => permissionChecker(STORAGE_LIST, user, () => storageService.getAllActive(token)),
+    dataStorage: (obj, args, { user, token }) => projectPermissionWrapper(args, STORAGE_LIST, user, () => storageService.getAllProjectActive(args.projectKey, token)),
```

Comment:
How come the `projectPermissionWrapper` takes `args` rather than destructuring the `args` like used to happen (`(obj, { projectKey }, { user, token })`? I know this change happened in another PR but I was curious as `projectPermissionWrapper` only uses the `projectKey` of `args`.

Reply:
I thought that was @gary-lloyd-tessella  's preference, but I may have got the wrong end of the stick?